src/Peachpie.AspNetCore.Web/RequestContextCore.cs

```diff
@@ -40,13 +40,15 @@ void IHttpPhpContext.SetHeader(string name, string value)
             }
 
             StringValues newitem = new StringValues(value);
-            //StringValues olditem;
-            //if (_httpctx.Response.Headers.TryGetValue(name, out olditem))
-            //{
-            //    newitem = StringValues.Concat(olditem, newitem);
-            //}
+            if (name.EqualsOrdinalIgnoreCase("set-cookie"))
```

Comment:
I wasn't sure why the original code was commented here so I only enabled it for set-cookie now.